### ISSUE TITLE
Bootstrap icon doesn't change when using sortReset function

### ISSUE THREAD

**Issue by reffu42** [Mottie/tablesorter#1361]
When using the `sortReset` trigger or calling the  `sortReset` function directly on a table, the sort resets, but the icon does not reset to the default unsorted icon. This can be confirmed in that the HTML for the header changes to indicate that the column is unsorted, but the icon (`<i>` tag) keeps whatever classes it had previously. This behavior differs from the `sortReset` setup option, which when set to `true` properly changes the icon to `bootstrap-icon-unsorted` on the third click. 

There's someone else with the same issue In this stackoverflow post: http://stackoverflow.com/questions/39950757/unsorted-icon-not-appearing-on-sort-reset-on-tablesorter

That issue was fixed by changing themes, but I'd prefer to keep the bootstrap theme for now.

I've also checked the second answer in that post, which suggests that the css file might not be loaded properly, but this isn't the case as the icon and other css work fine.

Right now I've gotten around this by triggering `sorton` and specifying a sort instead of using `sortReset`,` but it's still an issue that could be confusing to some.

**Comment by Mottie** [comment]
Hi @reffu42!

Thanks for reporting this problem! I am able to reproduce this error, and I will have a fix for it available soon.

**Comment by Mottie** [comment]
So after some investigation, I found that there wasn't a problem if you triggered the sort reset first:

```js
$('table').trigger('sortReset').trigger('filterReset');
```

But reversing the order causes the sortReset trigger to skip reapplying the widgets because it was still processing the filterReset widget reapplication.

I added a `setTimeout`, I know it's not ideal, to allow time for processing.

I think it might be a better idea to add a "resetSortAndFilter" method that combines the two methods so that you won't need to trigger multiple events.
